### DISCUSSION TITLE
jbovlaste 内の gismu 説明文の改定案について

### DISCUSSION THREAD

**Discussion by qq542vev** [ponjbogri/jvs_ja#10] [General]
coi xendo jboprenu

前々から jbovlaste 内の単語説明文ついて、単語ごとに微妙な表記のゆれがあり、改定したいと思っていたのですが、ひとまず基本 gismu のみで改定案を作成しました。

 * [改定前](https://gist.github.com/qq542vev/64fd8d342bb00a47581035ab6050693d#file-jicmu_gismu_se_galfi-xml)
 * [改定後](https://gist.github.com/qq542vev/64fd8d342bb00a47581035ab6050693d#file-jicmu_gismu_te_galfi-xml)

主な変更点は以下のとおりです。

 * `<notes>` 内の大意を `<glossword>` に移動。
 * `<notes>` 内の各項目を構造化。

   各項目は補足事項、例文、読み方、語呂合わせ、関連語の順で記述し、`  `(ダブルスペース)で区切る。項目内の細目は `, ` で区切る。
 * 全角記号を半角記号に変換。
 * 誤字脱字の修正。

変更の規模が大きいので、確認のために一度こちらに投稿しました。当然ながらこの上記の表記方法が完璧ではありませんし、さらにより良い変更案もあるかと思われます。もしご意見などがありましたら、投稿いただければ幸いです。

もし異論などがなければ、実際の変更作業に取り掛かるかもしれません。

**Comment by niftg** [comment]
https://gist.github.com/niftg/b27761f36a0e6ac3991a68146f76d0cc さしあたり現状の案で比較用diffファイルを作ってみました
多分問題ない気はしますが、全体を俯瞰したらまたコメント申し上げます

**Comment by niftg** [comment]
まだbratuのあたりまでしか見てませんがコメントしてみます

> * `<notes>` 内の大意を `<glossword>` に移動。

.i'e

> * `<notes>` 内の各項目を構造化。
>   各項目は補足事項、例文、読み方、語呂合わせ、関連語の順で記述し、`  `(ダブルスペース)で区切る。項目内の細目は `, ` で区切る。

スペースはダブルにしてもページの表示上は一つ分にしかならない気がするのでいまいちな気がしますが、構造化の目論見自体は悪くないと感じます

> * 全角記号を半角記号に変換。

これは全体にわたる変化ですが、現状そこまで急いでやることもないかなと感じます（好みもありますが、見映え的に、変に間が空いてしまったりするので）

> * 誤字脱字の修正。

.i'e。音素配列論的な変化への対応もなされてる感じですね（`brioco`→`bri'oco`mu'a)

引き続き眺めてみます

**Reply by qq542vev** [reply]
ご返信ありがとうございます。当方の思いつきのような提案にも関わらず、細かくチェックいただきありがとうございます。

> スペースはダブルにしてもページの表示上は一つ分にしかならない気がするのでいまいちな気がしますが、構造化の目論見自体は悪くないと感じます

構造化について、区切り文字のダブルスペースは、仰る通り表示上は一つ分なのですが、これ以外に有用な区切り文字を見つけるのが、中々に難しいです。

一般的な ASCII に収録されている記号ですと、どうしても必要となり、各項目内に現れる可能性があるような気が致します。一方でダブルスペースが現れることは(ミスタイプなどを除けば)非常に少ないのではないかと思います。

ただ区切り文字に ASCII 文字以外の非常に特殊な使う等々も考えられます。

> これは全体にわたる変化ですが、現状そこまで急いでやることもないかなと感じます（好みもありますが、見映え的に、変に間が空いてしまったりするので）

完全な好みの問題なのですが、説明文が縦書きならば全角、横書きならば半角が望ましいだろうと思い(**明確な根拠なし**)、一切合切変換してしまったので、仰る通りレイアウト上でおかしな部分はあるかと思います。

今回の改定案の XML は *[rolgimste-to-stura-sai-toi.xml](https://github.com/qq542vev/jihu-laho-zoi-pjcg-gimste-zoi/blob/main/valsi-liste/rolgimste-to-stura-sai-toi.xml)* から作成したのですが、余り手を加えていない *[rolgimste-to-stura-ruhe-toi.xml](https://github.com/qq542vev/jihu-laho-zoi-pjcg-gimste-zoi/blob/main/valsi-liste/rolgimste-to-stura-ruhe-toi.xml)* を改定案に利用すれば、全角記号のままかと思います。

当方の意見ばかり並べたてて申し訳ございません。にふさんの方でも、コメントやさらなる発見などがありましたら、ご報告いただければ幸いでございます。

**Reply by niftg** [reply]
je'e .io
約物の用法やその他の細部の体裁を調整･統一するのは、多分一筋縄で行かない部分もあるかと存じますし、優先度的には一旦傍に置いとくでもいいかなと私は感じます(pa'enai)

先の“主な変更点”について個人的に(あくまで作業の規模と効果のバランスを鑑みてですが)優先順位をつけるとしたら

1. glossword / 誤字脱字修正
2. “構造化”
3. 全角半角

みたいな感じですかね

ただ、主ではないもののチェックしておきたい他の変更点もある気がしたので、引き続き比較してみます.ai

**Reply by qq542vev** [reply]
返信ありがとうございます。引き続きチェックを行ったいただいているとのことで、誠に感謝です。

前述の投稿で書き忘れてしまったのですが、元のファイルと改定案の diff ファイルを作成してくださり、ありがとうございます。比較の際にとても見やすいです。

約物の件については了解致しました。一先ず体裁を元のままでということでしたので、*[rolgimste-to-stura-ruhe-toi.xml](https://github.com/qq542vev/jihu-laho-zoi-pjcg-gimste-zoi/blob/main/valsi-liste/rolgimste-to-stura-ruhe-toi.xml)* をベースにした改定案も作成してみました。

<https://gist.githubusercontent.com/qq542vev/64fd8d342bb00a47581035ab6050693d/raw/8610e153e1b69d991ea469e0779ffc7fa4597e85/xe_karbi_zohe_xire.diff>

記号などを変換していないため、大体原文のままだと思います。ただし `<notes>` 要素内でのダブルスペースによる構造化は行われています。最初の改定案であったような、誤字脱字の修整はありません。

*[rolgimste-to-stura-sai-toi.xml](https://github.com/qq542vev/jihu-laho-zoi-pjcg-gimste-zoi/blob/main/valsi-liste/rolgimste-to-stura-sai-toi.xml)* での誤字脱字の修整点を、今回の改定案にも反映できればよいのですが、何処をどのように修整したかを全く覚えていません。それ故改定案に反映させるのが難しいです。(泣)
